agilent_infiniium_scope.py

```diff
@@ -72,319 +72,350 @@ def channel_infoNONEXISTANT(self, c, channel):
         Tuple of (probeAtten, termination, scale, position, coupling, bwLimit, invert, units)
         """
         raise Exception('Not yet implemented')
-        #NOTES
-        #The scope's response to 'CH<x>?' is a string of format
-        #'1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
-        #These strings represent respectively,
-        #probeAttenuation;termination;vertScale;vertPosition;coupling;bandwidthLimit;invert;vertUnit
+        """
+        NOTES
+        The scope's response to 'CH<x>?' is a string of format
+        '1.0E1;1.0E1;2.5E1;0.0E0;DC;OFF;OFF;"V"'
+        These strings represent respectively,
+        probeAttenuation;termination;vertScale;vertPosition;coupling;
+        bandwidthLimit;invert;vertUnit
```

Comment:
I've fixed the # for the comments, but I do not know about the unpacking...

In fact, I dont think that the CH{}? command even exists for the Infiniium series of scopes.  I'm going to guess this was copied from another scope server and never implemented.  For these scopes this would be a series of individual commands.  Probably we should kill this function and we could add in different functions for each of these parameters, or just leave it out.